contracts/interfaces/ISecurityToken.sol

```diff
@@ -33,14 +33,14 @@ interface ISecurityToken {
      * @notice Burn function used to burn the securityToken
      * @param _value No. of tokens that get burned
      */
-    function burn(uint256 _value) external returns (bool success);
+    function burn(uint256 _value) external;
```

Comment:
agree with @maxsam4

Reply:
Since burn (and burnFrom) can only be used from a module, I am going to leave without the return statement. IMO the best semantics for these types of operations should be that they revert on failure. transfer / transferFrom are exceptions in order to meet ERC20 spec.